main/src/ca/uwaterloo/flix/language/dbg/PrettyPrinter.scala

```diff
@@ -0,0 +1,32 @@
+package ca.uwaterloo.flix.language.dbg
```

Comment:
Add copyright header.

Reply:
It is a good point. But I think for now we should tie it to lsp-- since that is how most people will use it.

Later, we can discuss a fmt command and then maybe it can get its own package somewhere. But for now I propose some api/lsp/provider/formatter/ package.